server/service/src/processors/transfer/shipment/README.md

```diff
@@ -20,4 +20,8 @@ When an outbound shipment is updated and an inbound shipment is already generate
 
 ## Same site transfer (both stores on same site)
 
-You may want to refer to [requisition trasfer docs](../requisition/README.md#same-site-transfer-both-stores-on-same-site) for example of how one instance of triggered processor can itself upsert records and process them in the next iteration
\ No newline at end of file
+You may want to refer to [requisition transfer docs](../requisition/README.md#same-site-transfer-both-stores-on-same-site) for example of how one instance of triggered processor can itself upsert records and process them in the next iteration
+
+## Merged names
+
+Stores have a direct FK to names via `store.name_id`. `changelog.name_id` is indirect, really being a FK to via `name_link.id`. As processors use the changelog to find records that require transfers to be generated, keep in mind that the change_logs need to selected for matching any `name_link.id` related to `store.name_id -> name_link.name_id, name_link.id -> changelog.name_id`. This will be more apparent when `changelog.name_id` is renamed to `changelog.name_link_id`.
```

Comment:
See comment above think we should do the column rename to changelog.name_link_id but only allow to query by `name_id`.

Writing hack: use one line per sentence. This makes reviewing and editing much easier because sentence changes stay within the same line...

Reply:
I removed because after your suggestion the comment was pretty much completely inaccurate haha. And probably not the best place to generally explain merging in OMS
https://github.com/msupply-foundation/open-msupply/pull/2811/commits/5312d97cdfb901629146d7fae7df93eb548fdf14